taichi/ir/statements.h

```diff
@@ -303,6 +303,28 @@ class GlobalPtrStmt : public Stmt {
   TI_DEFINE_ACCEPT_AND_CLONE
 };
 
+/**
+ * An accessing tensor element operation.
+ */
+class GlobalTensorElementStmt : public Stmt {
+ public:
+  Stmt *origin, *offset;
+  bool is_bit_vectorized;  // TODO: remove this field
```

Comment:
We don't need `is_bit_vectorized`?

Reply:
https://github.com/taichi-dev/taichi/pull/2543/commits/6922a516e0dcccd02be02ec07f3a8fa6f5ad371c